packages/bootstrap4-theme/gulpfile.js

```diff
@@ -127,26 +127,6 @@ gulp.task('scripts', function () {
     .pipe(gulp.dest(paths.dist + '/js'));
 });
 
-// Copy all master assets from the design-tokens library
-gulp.task('copy-assets', function (done) {
-  // Copy the master ASU images from the design-token
-  gulp
-    .src(`${paths.node}/@asu/design-tokens/build/assets/img/**/*`)
-    .pipe(gulp.dest('./src/img'));
```

Comment:
@scott-williams-az With the `design-tokens` deprecation, I was double-checking the images copied here were still in place for the repo, and looks like according to lines 77-85 as well as a test build I did locally, that is the case.  Confirming you came to the same conclusion.

Reply:
@scott-williams-az Also conflicts to be resolved. :P